docs/api/cypress-api/custom-commands.mdx

```diff
@@ -25,6 +25,19 @@ in the
 
 :::
 
+:::caution
```

Comment:
can we move above the info banner & just add to the text without the banner at all? Doesn't seem worthy calling out so loudly since this is what the doc is about

Reply:
Dig it